### PR TITLE
Adding function for instructions multiprocessing

### PR DESCRIPTION
**Pull request by Amit-Petavue**

### CODE REVIEW

File: benchmark_scripts/amazon-bedrock.py [hunk 1]
```diff
@@ -146,12 +149,48 @@ def run_queries_on_bedrock(
             )
 
 
-def run_inferences(args: Dict, model_instructions: Dict) -> None:
+async def multi_process(instruction_size,datasets_info, model_name, args):
```
Comment (Siddarth-Petavue) [inline benchmark_scripts/amazon-bedrock.py:152]: Add input and output data types

File: benchmark_scripts/amazon-bedrock.py [hunk 2]
```diff
@@ -146,12 +149,48 @@ def run_queries_on_bedrock(
...
+            dataset_length,
+        )
+        generate_gold_file(gold_file_list, model_file_path)
+        loop_end_time = datetime.now()
+        total_secs = (loop_end_time - loop_start_time).total_seconds()
+        print(
+            f"Time taken for {dataset_length} records: {get_elapsed_time(total_secs)}"
+        )
+
+def async_wrapper(instruction_size,datasets_info, model_name, args):
```
Comment (Siddarth-Petavue) [inline benchmark_scripts/amazon-bedrock.py:183]: move this to common functions file, dont duplicate same logic in all env

File: benchmark_scripts/amazon-bedrock.py [hunk 3]
```diff
@@ -56,7 +59,7 @@ def run_queries_on_bedrock(
     instruction_size: int,
     dataset_length: int,
 ) -> None:
-    for context, question, hardness in total_user_query:
+    for context, question, hardness in total_user_query[0:1]:
```
[the remark this answers is not in the document]
Comment (Amit-Petavue) [reply benchmark_scripts/amazon-bedrock.py:62]: done

File: benchmark_scripts/amazon-bedrock.py (hunk 2 above)
Comment (Amit-Petavue) [reply benchmark_scripts/amazon-bedrock.py:183]: Done

**Pull request merged**